handler/src/main/java/io/netty/handler/ssl/SslUtils.java

```diff
@@ -101,8 +106,26 @@
     static final String[] DEFAULT_TLSV13_CIPHER_SUITES;
     static final String[] TLSV13_CIPHER_SUITES = { "TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384" };
 
+    private static final boolean TLSV1_3_SUPPORTED;
+
     static {
-        if (PlatformDependent.javaVersion() >= 11) {
+        boolean tlsv13Supported = false;
+        try {
+            SSLContext context = SSLContext.getInstance("TLS", "SunJSSE");
```

Comment:
Why get SunJSSE explicitly? I'd understand if we got SunJSSE explicitly elsewhere, but it doesn't seem we do. If we're going to use the default implementation of "TLS" it seems we should check that one here. Otherwise there's lots of similar-but-not-quite-the-same checks we could do, like `Security.getProviders("SSLContext.TLSv1.3") != null` to see if any provider supports TLSv1.3.

Reply:
Somewhat agree here.   If a user has bouncy castle as their provider this may not match.

Reply:
good point... let me fix this